ai-avatar-demo/services/llm_service.py

```
@@ -0,0 +1,58 @@
+            stream = await self.client.chat.completions.create(
+                model="minimax/minimax-m2",
+                messages=cast(Any, openai_messages),
+                stream=True,
+                temperature=0.8,
+                max_tokens=2048,
+            )
+
+            async for chunk in stream:
+                if chunk.choices and chunk.choices[0].delta.content:
+                    yield chunk.choices[0].delta.content
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)